packages/ketchup/src/components/kup-echart/kup-echart.tsx

```diff
@@ -471,59 +477,76 @@ export class KupEchart {
     }
 
     #bubbleChart() {
-        const y = this.#createY(),
+        const x = this.#createX(),
+            y = this.#createY(),
             data = [],
             temp = [],
             legend = {},
             series = [];
-        let year = [];
-
-        const content = this.data.columns.map((data) => data.title);
+        let groups = [];
+        const defaultPointSize = '1000000000';
```

Comment:
Can you write a comment in the code to explain the reasoning behind this number for easier understanding?

Reply:
I added a comment and I'll try to explain here more on that.
In a nutshell, a defaultSize was added to correctly handle cases where a column for Size is not present in `data.columns`. 

The main problem here is incoherence in input data among showcase and webup.
- In Ketchup Showcase, mocked data for Bubble Graph provide 5 columns [x, y, size, label, group]; the last one was previously referenced in codebase as "year" but is actually the group we can see in legend.
- Data received in Webup for the same Graph Type has a different structure, providing 4 cols [label, x, y, group] 

Current implementation handles both these data formats.
We assumed input data in webup was correct, so this solution has the same behavior of legacy graph in jsf where, for the same data, bubbles have the same size. 
We can surely improve `#bubbleChart()` by conveying on the structure of input data.